templates/e2e-llm-workflows/deploy/jobs/ft.yaml

```diff
@@ -1,5 +1,5 @@
 name: e2e-llm-workflows
 entrypoint: llmforge anyscale finetune configs/training/lora/llama-3-8b.yaml
-image_uri: localhost:5555/anyscale/llm-forge:0.5.3
+image_uri: localhost:5555/anyscale/llm-forge:0.5.4
```

Comment:
does this have to be bumped again later to `0.5.5`?

Reply:
doesn't have to be right? atleast in the short term it's not necessary.

Reply:
sgtm